docs/source/collect/helm.md

```diff
@@ -87,7 +102,8 @@ When this collector is executed, it will include the following files in a suppor
 			{
 				"revision": "1",
 				"date": "2023-08-25 11:20:05.153483 +1200 NZST",
-				"status": "deployed"
+				"status": "deployed",
+        "values": {},
```

Comment:
I would be good if you added actual values instead of an empty object. You can collect values from https://github.com/replicatedhq/troubleshoot/blob/main/test/e2e/support-bundle/testdata/charts/nginx-15.2.0.tgz helm chart for example.